### PR TITLE
Fix/exercise 3.1 ur repository update

### PR DESCRIPTION
**Pull request by jrgnicho**
This PR updates the exercise 3.1 xacro and the documentation for it as well.  Main change is the use of the new ur repositories

### CODE REVIEW

**Comment by JeremyZoss** [comment]
Why did you change base_frame from world to table?  Shouldn't be required for Ex3.1 or future exercises...

Text before the git clone instructions still references `jdlangs` repo.  Should update to be more generic and remove the obsolete reference.

To get UR_ROS2_Driver to build, I also needed to install ros-foxy-ros2-control and ros-foxy-ros2-controllers.  Also had to change the branch on UR_ROS2_Driver repo from "main" to "foxy".

Prefer not to add unnecessary args to workcell xacro.  Not needed and just complicates this example.  I suggest removing all `<xacro:arg>` lines except maybe `ur_type`.  Just set their "default" values directly in the call to `<xacro:ur_robot>`.

Installation of Universal_Robots_ROS2_Driver dependencies could probably wait until a future exercise, but are just as easily done now.  Might be good to add explanation in exercise text for why we're installing the dependencies (and only a subset, at that).

**Comment by JeremyZoss** [comment]
This worked for me:
```
  <!-- ur arm instantiation -->
  <xacro:arg name="ur_type" default="ur5"/>
  <xacro:arg name="use_fake_hardware" default="true"/>
  <xacro:arg name="fake_sensor_commands" default="true"/>
  <xacro:ur_robot
      prefix=""
      joint_limits_parameters_file="$(find ur_description)/config/$(arg ur_type)/joint_limits.yaml"
      kinematics_parameters_file="$(find ur_description)/config/$(arg ur_type)/default_kinematics.yaml"
      physical_parameters_file="$(find ur_description)/config/$(arg ur_type)/physical_parameters.yaml"
      visual_parameters_file="$(find ur_description)/config/$(arg ur_type)/visual_parameters.yaml"/>
```

**Comment by jrgnicho** [comment]
@JeremyZoss 
- Good catch on the frame name, I'm not sure how that change made its way there
- Agree with updating the references to specific repos and modifying the instructions to make it more generic
- After cloning the UR repos we should include a rosdep step to install the ros2-control dependencies
- I'm good with using as many default arguments in the ur xacro in order to reduce complexity.
- I feel like having an extra step in this exercise to install the universal ROS2 driver dependencies isn't that big of a stretch, in fact it is necessary as the fake robot driver is provided by a ros2_control package and a rosdep install step would get this dependency

**Review by JeremyZoss — APPROVED** [review]
I've applied a patch to address my notes above.  It simplifies the xacro and launch file and cleans up some of the exercise text.  I did not follow Jorge's suggestion to add a rosdep call... personally, I think this adds potential extra complexity.  Instead, I've already added the required ros2_control packages to the system setup script (see #361).  The current state looks good to me.

File: gh_pages/_source/session3/ros2/1-Workcell-XACRO.md [hunk 1]
```diff
@@ -70,67 +82,45 @@ Specifically, you will need to:
  1. Create a new `urdf.launch.py` file (in the `myworkcell_support` package) to load the URDF model and (optionally) display it in rviz. The launch file starts with several utility functions that are useful for assisting the launch process. This particular file uses `get_package_file` to get the path of the `workcell.urdf.xacro` file you've created, and `run_xacro` to send it to the `xacro` program, generating a plain URDF file. This URDF file is then passed to a `robot_state_publisher` node as an argument just as was done manually in the previous exercise.
```
Comment (jdlangs) [inline gh_pages/_source/session3/ros2/1-Workcell-XACRO.md:82]: Leaving a note for followup: text still references `run_xacro` which isn't used anymore

**Review by jdlangs — APPROVED** [review]
Looks good. I'll merge now for exercise syncing purposes even though there's a least one more minor edit to do.

File: gh_pages/_source/session3/ros2/1-Workcell-XACRO.md (hunk 1 above)
Comment (JeremyZoss) [reply gh_pages/_source/session3/ros2/1-Workcell-XACRO.md:82]: Whoops.  Missed that.  Sorry.  I'll fix in an upcoming edit.  Thx for merging.

**Pull request merged**